web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -1,51 +1,47 @@
+import { ChecklistTag } from "@/components/ChecklistTag";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
 import { Icon } from "@/components/njwds/Icon";
+import { getAgencyText } from "@/components/TaskElement";
 import { getMergedConfig } from "@/contexts/configContext";
 import { useUserData } from "@/lib/data-hooks/useUserData";
 import analytics from "@/lib/utils/analytics";
 import { LicenseStatus, LicenseStatusItem } from "@businessnjgovnavigator/shared/";
-import { ReactElement, useEffect, useState } from "react";
+import { ReactElement } from "react";
 
 interface Props {
   items: LicenseStatusItem[];
   status: LicenseStatus;
+  agency: string;
```

Comment:
pebble - if both these props are going to be passed in, why not pass in the entire task?